### ISSUE TITLE
Question for new model experiment: are count data in the compendium raw or normalized?

### ISSUE THREAD

**Issue by taylorreiter** [greenelab/generic-expression-patterns#107]
After reading through the documentation, I'm a little unclear whether the input compendium should be normalized at all prior to input. I think it should be raw (number of reads mapped to a transcript [`NumReads` for salmon]), but I wanted to make sure that this was true. 

If it's not true, what is your recommendation for how to preprocess a compendium before using the notebook [1_process_and_train_vae.ipynb](https://github.com/greenelab/generic-expression-patterns/blob/master/new_model_experiment/1_process_and_train_vae.ipynb)?

**Comment by taylorreiter** [comment]
The normalize function will take care of normalization, so the input should be raw counts. 

In other experiments, Alex used:
+ microarray inputs which were already log2 scaled
+ recount2 were scaled counts (which was sort of like raw counts but is better described in documentation)

features need to be in the same range to train properly. Reason to 0-1 scale everything. In the past without that, the training curves didn't train -- the loss functions kept bopping around. 

0-1, at least in these scales, using cross entropy loss. Taking log of data. Most scraped from greg long ago :)

**Comment by ajlee21** [comment]
Addressed in https://github.com/greenelab/sophie/pull/3
